osh/hub/osh_xmlrpc/worker.py

```diff
@@ -243,3 +245,17 @@ def interrupt_tasks(request, task_list):
         fail_scan(request, sb.scan.id, 'Task was interrupted')
 
     return response
+
+
+@validate_worker
+def open_task(request, task_id):
+    response = kobo_open_task(request, task_id)
+
+    if settings.ENABLE_THROWAWAY_WORKERS:
+        task = Task.objects.get(id=task_id)
+        # Set the worker load to 0. This should avoid getting any new tasks assigned the worker.
+        # Each worker allocated through resalloc should be destroyed once a task completes.
+        task.worker.max_load = 0
+        task.worker.save()
```

Comment:
We agreed with @kdudka to avoid mixing static and dynamically allocated workers. Please see [this comment](https://github.com/openscanhub/openscanhub/pull/234#discussion_r1513029712).